requirements/base.txt

```diff
@@ -12,5 +12,5 @@ python-dateutil~=2.6, <2.8.1
 requests==2.22.0
 serverlessrepo==0.1.9
 aws_lambda_builders==0.5.0
-# https://github.com/mhammond/pywin32/issues/1439
-pywin32 < 226; sys_platform == 'win32'
\ No newline at end of file
+pywin32 < 226; sys_platform == 'win32'
```

Comment:
Can you add the GH Issue link back? It will useful to track when the issue is fixed

Reply:
yes, lost it in a rebase conflict.